ml-notebook/packages.txt

```diff
@@ -45,8 +45,7 @@ aws-c-sdkutils==0.1.16
 aws-checksums==0.1.18
 aws-crt-cpp==0.26.12
 aws-sdk-cpp==1.11.329
-awscli==2.17.0
-awscrt==0.20.10
+awscli==1.32.106
```

Comment:
https://github.com/conda-forge/awscli2-feedstock/pull/39

Reply:
Hmm, that python312 migration for awscli v2 doesn't look very active. Do we want to wait for that, or go with this awscli downgrade to v1?

Reply:
i think downgrading is fine. i've still yet to see a situation that *requires* v2 (https://docs.aws.amazon.com/cli/latest/userguide/cliv2-migration-changes.html)... As long as `aws s3 ls` works I'm happy :)

Reply:
Yeah, I don't do much fancy stuff either besides `aws s3 ls` and `aws s3 cp` :joy:. We'll hear soon enough if people have issues (and can let them pin to pangeo-docker-images `2024.06.23`:stuck_out_tongue_closed_eyes:)